day01/index.js

```diff
@@ -0,0 +1,45 @@
+const { pipe, reduce, add } = require('ramda');
+const convertInputToArr = require('./toArray');
+const fs = require('fs');
+const path = require('path');
+const trace = require('../lib/trace');
```

Comment:
I love that you were able to write a helper to assist with getting this challenge done! Super awesome.

Now that you're ready to merge this in, you can (and probably should) remove this line, as you're no longer using it in this code. I say "can remove" because...you're just not using it anymore. I say "should" because a [linter](https://gomakethings.com/javascript-linters/) would catch that you've defined this variable, but you're not referencing it anywhere, and you'd have to remove this reference before your PR could be merged.

Reply:
Good catch, I had meant to remove that.